vic/drivers/cesm/src/vic_force.c

```diff
@@ -205,6 +205,10 @@ vic_force(void)
         for (j = 0; j < NF; j++) {
             // vapor pressure deficit
             force[i].vpd[j] = svp(force[i].air_temp[j]) - force[i].vp[j];
+	    if (force[i].vpd[j] < 0) {
+		force[i].vpd[j] = 0;
+		force[i].vp[j] = svp(force[i].air_temp[j]);
+	    }
```

Comment:
Add `log_warn()` when this happens with info about the reset (display the value of `vpd` before the reset and say it was rest to 0).